### PR TITLE
COMMERCE-4427 Make sku contributor options required while editing a sku

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
@igorbeslic FIY

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-ddm-impl/src/main/java/com/liferay/commerce/product/ddm/internal/DDMHelperImpl.java [hunk 1]
```diff
@@ -299,7 +299,7 @@ private String _createDDMFormRuleOutputMapping(DDMForm ddmForm) {
 
 	private DDMForm _getDDMForm(
 		Locale locale, boolean ignoreSKUCombinations, boolean optional,
-		boolean publicStore,
+		boolean publicStore, boolean required,
```
Comment (igorbeslic) [inline modules/apps/commerce/commerce-product-ddm-impl/src/main/java/com/liferay/commerce/product/ddm/internal/DDMHelperImpl.java:302]: @Commando18 This is not good fix.. Parameter optional is one that tells if option is required or optional. You added unnecessary  redundancy here.

Comment (Commando18) [reply modules/apps/commerce/commerce-product-ddm-impl/src/main/java/com/liferay/commerce/product/ddm/internal/DDMHelperImpl.java:302]: Ok, thanks, I will use that one then

File: modules/apps/commerce/commerce-product-ddm-impl/src/main/java/com/liferay/commerce/product/ddm/internal/DDMHelperImpl.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ public DDMForm getCPAttachmentFileEntryDDMForm(
 			cpDefinitionOptionRelCPDefinitionOptionValueRels) {
 
 		return _getDDMForm(
-			locale, false, true, false,
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-ddm-impl/src/main/java/com/liferay/commerce/product/ddm/internal/DDMHelperImpl.java:84]: @Commando18 considering Igor comment I believe this should be left to true

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**